common/test/acceptance/tests/video/test_video_module.py

```diff
@@ -1205,13 +1205,21 @@ def setUp(self):
             super(LMSVideoModuleA11yTest, self).setUp()
 
     def test_video_player_a11y(self):
+        # load transcripts so we can test skipping to
+        self.assets.extend(['chinese_transcripts.srt', 'subs_3_yD_cEKoCk.srt.sjson'])
+        data = {'transcripts': {"zh": "chinese_transcripts.srt"}, 'sub': '3_yD_cEKoCk'}
+        self.metadata = self.metadata_for_mode('youtube', additional_data=data)
+
+        # go to video
         self.navigate_to_video()
+        self.video.show_captions()
 
-        # Limit the scope of the audit to the video player only.
-        self.video.a11y_audit.config.set_scope(include=["div.video"])
+        # limit the scope of the audit to the video player only.
+        self.video.a11y_audit.config.set_scope(
+            include=["div.video"],
+            exclude=["a.ui-slider-handle"]
+        )
         self.video.a11y_audit.config.set_rules({
```

Comment:
Same thing here.  Since it's no longer really doing anything, just remove the whole `self.video.a11y_audit.config.set_rules` call.